requirements.txt

```diff
@@ -1,101 +1,5 @@
-#ckan extensions
```

Comment:
Why did you delete this entire file?

Reply:
It's not fully deleted. Since we remove all extensions (to start adding it one by one) I just restart the file with ckanext-harvest only.
Base docker container from OKF already include basic requirements from CKAN @akariv.

Reply:
I see it also contains pinned dependencies for ckan and the extensions, not just the extensions themselves.

Reply:
The `openknowledge/ckan-base:2.8` base image we are using already includes the requirements file from CKAN 2.8. That's why we don't need this reqs here. This branch run CKAN and also is able to harvest.

We can also go an step back and use the base container in which we [manage requirements](https://github.com/okfn/docker-ckan/blob/master/ckan-base/2.8/Dockerfile#L66) also for ckan.